orc8r/gateway/python/magma/ctraced/trace_manager.py

```diff
@@ -0,0 +1,104 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+import os
+import pathlib
+import subprocess
+
+from subprocess import SubprocessError
+
+TRACE_FILE_NAME = "call_trace.pcap"
+
+class TraceManager:
```

Comment:
AGW can have multiple interfaces. In fact if you want to capture traffic coming from eNb and leaving AGW you may need to do it through the GTP bridge and some other ethX.

So maybe it would be good to create Trace object. Then TraceManager would manage many Trace objects. TraceManager could make sure to start/stop all the traces at the same time and make sure only one trace is running per interface